cmd/logcli/main.go

```diff
@@ -109,13 +109,13 @@ func newQueryClient(app *kingpin.Application) *client.Client {
 		return nil
 	}
 
-	app.Flag("addr", "Server address.").Default("https://logs-us-west1.grafana.net").Envar("GRAFANA_ADDR").Action(addressAction).StringVar(&client.Address)
-	app.Flag("username", "Username for HTTP basic auth.").Default("").Envar("GRAFANA_USERNAME").StringVar(&client.Username)
-	app.Flag("password", "Password for HTTP basic auth.").Default("").Envar("GRAFANA_PASSWORD").StringVar(&client.Password)
-	app.Flag("ca-cert", "Path to the server Certificate Authority.").Default("").Envar("LOKI_CA_CERT_PATH").StringVar(&client.TLSConfig.CAFile)
+	app.Flag("addr", "Server address. Can also be set using LOKI_ADDR ENV VAR.").Default("http://localhost:3100").Envar("LOKI_ADDR").Action(addressAction).StringVar(&client.Address)
```

Comment:
No need to capitalize "env var". Also, should be "environment variable", but that is perhaps too long.